client/js/lounge.js

```diff
@@ -734,7 +759,7 @@ $(function() {
 	});
 
 	var windows = $("#windows");
-	var forms = $("#sign-in, #connect");
+	var forms = $("#sign-in, #connect, #change-password");
```

Comment:
AFAICT this is used to wire-up the on-submit event handlers on each of the three elements with those IDs, which happen to be `<form>` elements (the elements are selected separately as there's commas between each ID, and therefore will return a JQuery object referencing each of the elements with IDs `sign-in`, `connect`, and `change-password`). The event handler we wire-up onto this resultant set of elements calls `event.preventDefault()` which stops the form submittal from reloading the page. The form is submitted through the websocket or XHR instead?